README.md

```diff
@@ -18,6 +18,43 @@ To dig straight into our documentation and get up and running, try:
 
 ## Usage
 
+All the directories here are required and will be available on production web servers. Any extra directories will not be available in production.
```

Comment:
Let's add a link to our public docs here for more information about what directories do what on the platform: https://docs.wpvip.com/technical-references/vip-codebase/